iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/MultiplexingClientTests.java

```diff
@@ -1369,7 +1369,9 @@ public void registerDeviceWithIncorrectCredentialsBeforeOpenThrowsOnOpen() throw
         boolean expectedExceptionThrown = false;
         try
         {
-            testInstance.multiplexingClient.open(true);
+            // Note that we are opting to not retry on open because the first attempt should result in the caught
+            // exception below.
+            testInstance.multiplexingClient.open(false);
```

Comment:
Shouldn't bad credentials cause the retry policy in place to fail immediately anyway?

Reply:
Good point. I'll look into this